lib/Models/Catalog/Esri/ArcGisPortalItemReference.ts

```diff
@@ -166,15 +166,16 @@ export default class ArcGisPortalItemReference extends AccessControlMixin(
       createStratumInstance(ArcGisPortalItemFormatTraits, {
         id: "WMS",
         formatRegex: "WMS",
-        urlRegex: "WMSServer",
+        urlRegex: "WMSServer|wms$",
         definition: {
-          type: "wms"
+          type: "wms-group"
```

Comment:
Would this change impact singular WMS layers matching the regex?

Reply:
I think it will become one member group.